ros/src/actuation/vehicles/packages/as/nodes/as_interface/as_interface.cpp

```diff
@@ -87,13 +151,13 @@ void PacmodInterface::callbackFromTwistCmd(const geometry_msgs::TwistStampedCons
   speed_mode.header = msg->header;
   speed_mode.mode = mode;
   speed_mode.speed = msg->twist.linear.x;
-  speed_mode.acceleration_limit = 3.0;
-  speed_mode.deceleration_limit = 3.0;
+  speed_mode.acceleration_limit = acceleration_limit_;
+  speed_mode.deceleration_limit = deceleration_limit_;
 
   module_comm_msgs::SteerMode steer_mode;
   steer_mode.header = msg->header;
   steer_mode.mode = mode;
-  double curvature = msg->twist.angular.z / msg->twist.linear.x;
+  double curvature = msg->twist.linear.x?msg->twist.angular.z / msg->twist.linear.x:0.0;
```

Comment:
This is a sketchy way to check for division by zero. I would define a minimum linear x instead

Reply:
@yk-fujii do you agree, will you fix?

Reply:
yep, I agree.
I had fixed  this. Please check it.